app/containers/GamePage/index.js

```diff
@@ -118,7 +117,7 @@ export default class GamePage extends Component {
         if (participants[i].key === data.key) {
           participants.splice(i, 1);
           const num = participants.length;
-          this.setState({ participants, players: num });
+          this.setState({ participants, totalPlayers: num });
```

Comment:
I see you changed players to totalPlayers but you aren't using totalPlayers anymore